tabpy/tabpy_server/app/util.py

```diff
@@ -51,42 +47,41 @@ def parse_pwd_file(pwd_file_name):
 
     credentials : dict
         Credentials from the file. Empty if succeeded is False.
-    '''
-    logger.info(f'Parsing passwords file {pwd_file_name}...')
+    """
+    logger.info(f"Parsing passwords file {pwd_file_name}...")
 
     if not os.path.isfile(pwd_file_name):
-        logger.critical(f'Passwords file {pwd_file_name} not found')
+        logger.critical(f"Passwords file {pwd_file_name} not found")
         return False, {}
 
     credentials = {}
     with open(pwd_file_name) as pwd_file:
-        pwd_file_reader = csv.reader(pwd_file, delimiter=' ')
+        pwd_file_reader = csv.reader(pwd_file, delimiter=" ")
         for row in pwd_file_reader:
             # skip empty lines
             if len(row) == 0:
                 continue
 
             # skip commented lines
-            if row[0][0] == '#':
+            if row[0][0] == "#":
                 continue
 
             if len(row) != 2:
-                logger.error(
-                    f'Incorrect entry "{row}" in password file')
+                logger.error(f'Incorrect entry "{row}" in password file')
                 return False, {}
 
             login = row[0].lower()
             if login in credentials:
                 logger.error(
-                    f'Multiple entries for username {login} '
-                    'in password file')
+                    f"Multiple entries for username {login} " "in password file"
```

Comment:
These 2 strings should be concatenated.

Reply:
Nice find. Yea this is one open issue with black when you first run on a code base

https://github.com/psf/black/issues/26